pandas/tests/groupby/test_groupby.py

```diff
@@ -361,6 +363,119 @@ def f3(x):
         df2.groupby("a").apply(f3)
 
 
+def test_groupby_indices_error():
+    # GH 26860
+    # Test if DataFrame Groupby builds gb.indices
+    dt = pd.to_datetime(["2018-01-01", "2018-02-01", "2018-03-01"])
+    df = DataFrame(
+        {
+            "a": Series(list("abc")),
+            "b": Series(dt, dtype="category"),
+            "c": pd.Categorical.from_codes([-1, 0, 1], categories=[0, 1]),
+        }
+    )
+
+    df.groupby(["a", "b"]).indices
```

Comment:
you want to test the result not just the absence of an error
use (here and elsewhere) the pattern:
```
result = ...
expected = ...
assert result == expected
```

Reply:
If I recall correctly, this test was requested since it directly tests the bug in the issue which was an exception being raised.  The other test is about the result.

Reply:
ok if the result is being tested below then we don't need this